src/components/cart/Cart.js

```diff
@@ -0,0 +1,57 @@
+// Libraries
+import PropTypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+// components
+import CartCheckout from "./components/cartCheckout";
+import CartItemsList from "./components/cartItemsList";
+
+// utility
+import getFormattedItemsString from "../../utility/getFormattedItemsString";
+
+// helpers
+import sumOfValues from "./helpers/sumOfValues";
+
+// css
+import "./cart.css";
+
+function Cart(props) {
+    const { quantities, foodList, handleClearCart, ...rest } = props;
```

Comment:
rest props are not required here, we mainly keep it in the case of reusable component